modules/_labs/panel/react/lib/Panel.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {colors, spacing, CanvasSpacingValue} from '@workday/canvas-kit-react-core';
+
+export interface PanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  padding?: CanvasSpacingValue;
+  openDirection?: PanelDirection;
```

Comment:
What are your thoughts on `SidePanel` being controllable but this one not?

Also, do you plan to update `SidePanel` to use this eventually?

Reply:
Considering that this component will probably we contained within another element, I'm ok with removing this prop.

Also, I'd love to create a a component that can be added to and build upon. So yes, I'd love for side panel to be build on the panel component. I was talking to @NicholasBoll and we discussed separating out the pieces of it, which is why I moved `PanelHeader` into its own component instead of having a super flexible single component.

Reply:
Oh I wasn't saying we should remove `openDirection`. I think this is a good way to call out which side the border, etc. should be on. I was just pointing out the fact that you can actually control whether `SidePanel` is open/closed where as this one you have to wrap with a component that does that for you. I wanted to get your thoughts on why we have them structured that way.

Reply:
Ah sorry, yeah misread that. If this component where manage being open/closed I'd say then yes, but my thought was, by leaving it out, it allows the consumer to manage if they want to put in something like layout, or control from where it opens, i suppose a little more flexible. As a base component i chose this approach, but something like side panel, it made sense to include that functionality. I'm open to adding it, especially maybe some animation? Thoughts?

Reply:
Makes sense - I just wanted to hear your thoughts!